SECURITY.md

```diff
@@ -1,7 +1,7 @@
-# Eiffel-community vulnerability and Security reporting
+# Eiffel-community vulnerability and security reporting and response
 
 ## Report a Vulnerability or Security issue
-We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers.
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [eiffel community security page](https://eiffel-community.github.io/security.html).
```

Comment:
Sorry if I have missed something, please correct me. Shouldn't we link to the pull request to create the github.io page in this request?

Reply:
i think that needs to be done once the process is agreed on.

Reply:
There is a similar sentence under the public disclosure paragraph. I don't think we need to have it both places. @eiffel-community/eiffel-community-maintainers while you are eagerly reviewing this ;), think about were do we want the link?

Reply:
I posted a change suggestion on how to solve this.